packages/web-runtime/src/pages/account.vue

```diff
@@ -73,19 +87,29 @@
 
 <script>
 import { mapGetters } from 'vuex'
+import EditPasswordModal from '../components/EditPasswordModal.vue'
+import { clientService } from 'web-pkg/src/services'
+
 export default {
   name: 'Personal',
+  components: {
+    EditPasswordModal
+  },
   data() {
     return {
       loadingGroups: true,
-      groups: []
+      groups: [],
+      editPasswordModalOpen: false
     }
   },
   computed: {
-    ...mapGetters(['user', 'configuration', 'getNavItemsByExtension', 'apps']),
+    ...mapGetters(['user', 'configuration', 'getNavItemsByExtension', 'apps', 'capabilities']),
     isAccountEditingEnabled() {
       return !this.apps.settings
     },
+    isChangePasswordEnabled() {
+      return this.capabilities.spaces?.enabled
```

Comment:
Please add a `FIXME` comment explaining that spaces.enabled is not the correct capability for this.